docker.go

```diff
@@ -26,6 +27,15 @@ var (
 		RunE:         runDockerPin,
 	}
 
+	dockerBaseCmd = &cobra.Command{
+		Use:          "resolve [base image]",
+		Example:      "resolve ubuntu:20.04",
+		Short:        "Prints the current digest of the given base image",
+		Args:         cobra.ExactArgs(1),
+		SilenceUsage: false,
```

Comment:
SilenceUsage should probably be true, because we don't want to print the usage in case we return an error from the implementation.